src/game.cpp

```diff
@@ -316,6 +316,85 @@ static inline u8 getNeighbors(v3s16 p, INodeDefManager *nodedef, ClientMap *map,
 	return neighbors;
 }
 
+namespace {
+class Interpolator

```

Comment:
The name is generic, but the scope is limited. The class exists to pull out the Interpolation state from the function itself (my first iteration had this all getPointedThing but it seemed to be a lot for the reader to hold in their head at once.) Given the name, and the usage, it seems to me to be pretty clear what it does.